zendesk/bin/komand_zendesk

```diff
@@ -7,7 +7,7 @@ from komand_zendesk import connection, actions, triggers
 Name = 'Zendesk'
 Vendor = 'rapid7'
 Version = '1.1.1'
-Description = 'The Zendesk plugin helps manage communication with customers. This plugin allows you to manage tickets and users in Zendesk of data'
+Description = 'Manage customer communications. Manage tickets and users in Zendesk data'
```

Comment:
```suggestion
Description = 'Regulate customer communications by managing tickets and users in Zendesk data'
```

Reply:
this should be altered in the spec and in the help and then run regenerate to fix it in this bin file